src/__tests__/WeeklySummary.test.js

```diff
@@ -9,19 +9,19 @@ const { store } = configureStore();
 const url = ENDPOINTS.USER_PROFILE(':userId');
 
 const weeklySummariesMockData = {
-  weeklySummariesCount: 1,
-  weeklySummaries: [{ _id: '1', dueDate: '1', summary: 'a' }],
-  mediaUrl: 'u',
+    weeklySummariesCount: 1,
+    weeklySummaries: [{ _id: '1', dueDate: '1', summary: 'a', uploadDate: '1' }],
+    mediaUrl: 'u',
 };
 
 const server = setupServer(
-  rest.get(url, (req, res, ctx) => res(ctx.json(weeklySummariesMockData), ctx.status(200))),
-  rest.get('*', (req, res, ctx) => {
-    console.error(
-      `Please add request handler for ${req.url.toString()} in your MSW server requests.`,
-    );
-    return res(ctx.status(500), ctx.json({ error: 'You must add request handler.' }));
-  }),
+    rest.get(url, (req, res, ctx) => res(ctx.json(weeklySummariesMockData), ctx.status(200))),
+    rest.get('*', (req, res, ctx) => {
+        console.error(
+            `Please add request handler for ${req.url.toString()} in your MSW server requests.`,
+        );
+        return res(ctx.status(500), ctx.json({ error: 'You must add request handler.' }));
+    }),
```

Comment:
Hi, I believe coding conventions for indentation is 2 spaces instead of 4 for JS due to callback hell. Please change this file's indentation from 4 spaces to 2.